### DISCUSSION TITLE
Windows Aero with XP Layout

### DISCUSSION THREAD

**Discussion by Doofus-Mc-Goofus** [Open-Shell/Open-Shell-Menu#1977] [Show and tell]
Surprised nobody has made one. Made an XP theme that is compatible with glass transparency. I would give the option to make the username field background more of an xp look, but Resource Hacker hates me and doesn't like my 32bit bmps. Few bugs to note:
Disabling user picture causes everything to break, centering username breaks username, large fonts don't work, and this theme is ONLY compatible with the two columns style. If I can get Resource Hacker to import my files I will make a v2.
![Preview](https://github.com/user-attachments/assets/12be4c27-b85d-420f-a45a-66bdb5947fbe)
[Luna+.zip](https://github.com/user-attachments/files/16733942/Luna%2B.zip)

**Comment by Doofus-Mc-Goofus** [comment]
Version 2 is in the works.
![V2preview](https://github.com/user-attachments/assets/da022ef5-6e16-4a83-a229-ca18650911b4)

**Reply by ijustatemyTV** [reply]
whens version two planned to come out?

**Reply by Doofus-Mc-Goofus** [reply]
Sadly never. I lost the files FOREVER ago and I don't really have the time or motivation to start again.
